src/Symfony/Component/Console/Application.php

```diff
@@ -1169,4 +1169,27 @@ private function splitStringByWidth($string, $width)
 
         return $lines;
     }
+
+    /**
+     * Returns all namespaces of the command name.
+     *
+     * @param string $name The full name of the command
+     *
+     * @return array The namespaces of the command
+     */
+    private function extractAllNamespaces($name)
+    {
+        $parts = explode(':', $name, -1);
```

Comment:
No worries, the case we are looking at is not a big deal at all. Just wanted to point out that any comment can be turned into a method name.

:+1: